src/App.tsx

```diff
@@ -1,10 +1,19 @@
-import React from "react";
+import Header from "./components/Header";
+import { getCurrentPageForUrl, getUpdateMenu } from "./utils";
+import headerData from "./headerData.json";
 
 function App() {
+  const { pathname } = window.location;
+  const { nav } = headerData;
+  // Recursively set isSelected property in menu items
+  const updatedNav = getUpdateMenu(pathname, nav);
+  // Match pathname with a set of predefined urls "urlsWebApp"
+  const currentPage = getCurrentPageForUrl(pathname);
+  const isInactiveSection = !currentPage || !currentPage.menu;
```

Comment:
updateNav y isInactiveSection podrian ser funciones importadas desde ./utils moviendo esta logica a ese file

Reply:
Estoy de acuerdo. Creé una pequeña PR para agregar estas sugerencias, gracias!

[updateNav and isInactiveSection moved](https://github.com/FromDoppler/doppler-menu-mfe/pull/27/files#diff-26ad4b834941d9b19ebf9db8082bd202aaf72ea0ddea85f5a8a0cb3c729cc6f2L8-L12)

Reply:
Estoy de acuerdo. Creé una pequeña PR para agregar estas sugerencias, gracias!

[updateNav and isInactiveSection moved](https://github.com/FromDoppler/doppler-menu-mfe/pull/27/files#diff-26ad4b834941d9b19ebf9db8082bd202aaf72ea0ddea85f5a8a0cb3c729cc6f2L8-L12)